src/main/java/seedu/address/logic/commands/FindCommand.java

```diff
@@ -4,20 +4,20 @@
 
 import seedu.address.commons.core.Messages;
 import seedu.address.model.Model;
-import seedu.address.model.person.NameContainsKeywordsPredicate;
+import seedu.address.model.module.NameContainsKeywordsPredicate;
 
 /**
- * Finds and lists all persons in address book whose name contains any of the argument keywords.
+ * Finds and lists all Modules in Module book whose name contains any of the argument keywords.
  * Keyword matching is case insensitive.
  */
```

Comment:
Since we are modifying the Find command to locate modules not yet tracked by the user, we will need to log an issue here